documentation/user_docs/docs/user_guide/Horace3_to_4.rst

```diff
@@ -0,0 +1,236 @@
+####################################
+Converting from Horace 3 to Horace 4
+####################################
+
+Horace 4 has been a major rewrite and refactoring effort on the code
+from Horace 3. This document outlines the breaking changes between
+versions as well as describing a fast way to convert existing scripts
+from Horace 3 to Horace 4.
+
+Projections
+-----------
+
+In Horace 4, projections have been expanded and reworked to be
+generalised for other shapes of cuts (e.g. spherical, cylindrical). As
+a result, the old-style projections, e.g.:
+
+.. code-block:: matlab
+
+   proj1 = struct('u', [1 0 0], 'v', [0 1 0]);
+   proj2 = projaxes([1 0 0], [0 1 0]);
+   proj3 = ortho_proj([1 0 0], [0 1 0]);
+
+   proj4.u = [1 0 0];
+   proj4.v = [0 1 0];
+
+have been deprecated and a warning will be issued if they are
+used. For old-style projections the conversion is as simple as
+replacing the ``struct``, ``projaxes`` or ``ortho_proj`` with ``line_proj``
+(c.f. :ref:`projections
+<manual/Cutting_data_of_interest_from_SQW_files_and_objects:Projection in more details>`).
+
+.. code-block:: matlab
+
+   proj1 = line_proj([1 0 0], [0 1 0]);
+   proj2 = line_proj([1 0 0], [0 1 0]);
+   proj3 = line_proj([1 0 0], [0 1 0]);
+   proj4 = line_proj([1 0 0], [0 1 0]);
+
+Symmetrisation
+--------------
+
+The function ``sqw.symmetrise_sqw`` has been significantly rewritten
+to generalise it and allow rotational symmetry reduction. As part of
+this rewrite, it now takes ``Symop`` s as arguments rather than a pair
+of vectors.
+
+To update e.g.:
+
+.. code-block:: matlab
+
+   symmetrised = win.symmetrise_sqw([1 0 0], [0 1 0], [1 1 1]);
+
+it simply becomes:
+
+.. code-block:: matlab
+
+   symmetrised = win.symmetrise_sqw(SymopReflection([1 0 0], [0 1 0], [1 1 1]);
+
+.. note::
+
+   The ``SymopReflection`` can also be constructed earlier and reused:
+
+   .. code-block:: matlab
+
+      sym = SymopReflection([1 0 0], [0 1 0], [1 1 1]);
+      symmetrised = win.symmetrise_sqw(sym);
+
+Passing a pair of vectors will still be accepted and be internally
+transformed into the correct ``Symop``, however, it is still best to
+switch.
+
+Crystal Alignment
+-----------------
+
+Crystal alignment in Horace 4 has been changed such that alignment is
+a two-stage process.
+
+When you call ``change_crystal`` to an ``sqw``, it will not transform
+the pixels immediately, but most operations will apply the transform
+whenever the pixels are used allowing you to e.g. plot the realigned
```

Comment:
This all was in Horace-3. The changes are in interface. I should probably write the change to interface. 

Or you can write it -- the interface in Horace-4 substantially simplified and follows Horace-3 interface for `rlu_corrections` matrix only. 

formally the interface in all cases is like the one used for `rlu_corrections` but `crystal_alignment_matrix` class is used instead of `rlu_corrections` matrix. The class contains all other alighment information instead of multiple parameters used in Horace-3

Reply:
yes, but you describe things which was there already in Horace-3
I've found the change Horace-3->Horace-4. 
Sample sript on `IDAAAS->HoraceExampleScripts`
Row 41
```
[rlu_corr,alatt,angdeg] = refine_crystal(rlu0, alatt, angdeg, bp,'fix_angdeg','fix_alatt_ratio'); 
```
should be 
```
rlu_corr = refine_crystal(rlu0, alatt, angdeg, bp,'fix_angdeg','fix_alatt_ratio'); 
```
and `rlu_corr` now is the class containing all changes. Not a matrix as in Horace-3

These are all changes from Horace-3 to Horace-4. 

More substantial issue is below